source/fe/fe_simplex_p.cc

```diff
@@ -168,8 +219,6 @@ namespace
   {
     const unsigned int dim = 2;
 
-    Assert(degree <= 2, ExcNotImplemented());
```

Comment:
Should this check for <=3 now? I guess this is checked elsewhere as well, so perhaps it's not necessary to duplicate the test.